content/millennium/dstu2/conformance/operation-definition.md

```diff
@@ -58,7 +58,7 @@ Authorization is not required.
 
 ### Headers
 
-<%= headers head: {'<a href="../#media-types">Accept</a>': 'application/json+fhir'} %>
+<%= headers head: {'<a href="../../#media-types">Accept</a>': 'application/json+fhir'} %>
```

Comment:
can do:
```suggestion
<%= headers head: {Accept: 'application/json+fhir'} %>
```
we only needed the whole thing previously since the defaulted ../../ didn't work for the page being a level above other resource pages

Reply:
Made the change here:  [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/2bcc448db093c8c5488198f21d9937fba354d60e#diff-9f99b6db0052e592a000d125a263da2aL61)